### DISCUSSION TITLE
File name for new posts

### DISCUSSION THREAD

**Discussion by JPMorissette** [MrGreensWorkshop/MrGreen-JekyllTheme#180] [How to] [answered]
Hello!
First of all, thank you so much for this! I'm new to Git and I'm currently trying to build my website using your template.

I have no idea if there is a reason why this is happening, but in my folder _posts, I managed to change a first post to test how everything works. I think I understand most of what I can change, but somehow I can't change the format of the markdown files. Is this normal?

Exemple:

2023-27-02-Test.markdown works fine

Test.markdown  doesn't show up on my Blog page.

Does this mean that I _need_ to use the date format in my file names for them to show up?

Thanks!

**Comment by MrGreensWorkshop** [comment, marked as answer]
Hi,

The site is generated with Jekyll. For more information about Jekyll posts, which explain the format of post file names, please check the [documentation](https://jekyllrb.com/docs/posts/).

These links may be of assistance to you in the future.

 - Watching [Mr. Green theme tutorials playlist](https://www.youtube.com/playlist?list=PLAymxPbYHgl-fFy5can7uZBMJtFWVcphD) on YouTube.
 - Searching the existing [Issues](https://github.com/MrGreensWorkshop/MrGreen-JekyllTheme/issues/).
 - Checking the existing [Discussions](https://github.com/MrGreensWorkshop/MrGreen-JekyllTheme/discussions/).
 - Checking the [jekyll docs](https://jekyllrb.com/docs/).
